sharding/client/client.go

```diff
@@ -30,17 +27,28 @@ const (
 	clientIdentifier = "geth" // Used to determine the ipc name.
 )
 
-// Client for Collator. Communicates to Geth node via JSON RPC.
-type collatorClient struct {
-	endpoint string             // Endpoint to JSON RPC
-	client   *ethclient.Client  // Ethereum RPC client.
-	keystore *keystore.KeyStore // Keystore containing the single signer
-	ctx      *cli.Context       // Command line context
-	smc      *contracts.SMC     // The deployed sharding management contract
+// General Client for Collator. Communicates to Geth node via JSON RPC.
```

Comment:
This is not only for collator, it's for proposer and future actors too

Reply:
Done